py_ecc/fields/optimized_field_elements.py

```diff
@@ -182,6 +182,15 @@ def __repr__(self: T_FQ) -> str:
     def __int__(self: T_FQ) -> int:
         return self.n
 
+    def sgn0(self: Type[T_FQ]) -> int:
+        if self.n == 0:
+            return 0
+            neg = -self
```

Comment:
looks like your indentation got a little off here.

Reply:
Thanks, it sure is!